web-local/src/hooks.client.ts

```diff
@@ -0,0 +1,20 @@
+import { dev } from "$app/environment";
+import {
+  projectInitialized,
+  runtime,
+} from "@rilldata/web-common/runtime-client/runtime-store";
+
+/** INITIALIZE RUNTIME STORE **/
+
+// When testing, we need to use the relative path to the server
+const HOST = dev ? "http://localhost:9009" : "";
+const INSTANCE_ID = "default";
+
+const runtimeInit = {
+  host: HOST,
+  instanceId: INSTANCE_ID,
+};
+
+runtime.set(runtimeInit);
+
+projectInitialized.init().catch(console.error);
```

Comment:
This (maybe counter-intuitively) leads to the wrong behavior. Run `./rill start new-project` to see. We want a new project to open to the `/welcome` page.

Reply:
This was somewhat unrelated. The list files query returns a 400 response if there's no project folder and our `fetchWrapper` rejects the promise when that happens. Wrapping the query in a try catch block solved the issue, though I would argue that we should change the `fetchWrapper` behavior since this is kind of in conflict with Svelte Query's stateful error approach.